src/components/screens/vacation-requests-screen.tsx

```diff
@@ -39,15 +40,37 @@ const VacationRequestsScreen = () => {
   const [vacationRequests, setVacationRequests] = useAtom(
     adminMode ? allVacationRequestsAtom : vacationRequestsAtom
   );
+  const [_displayedVacationRequests, setDisplayedVacationRequests] = useAtom(displayedVacationRequestsAtom)
```

Comment:
As you are only setting here perhaps the useSetAtom could be useful. You should find other uses of this within the project and of course feel free to read the docs.